### PR TITLE
add superclass to 'hypertrophic cardiomyopathy 9'

### LINKED ISSUE #7346: hypertrophic cardiomyopathy 9
The condition hypertrophic cardiomyopathy 9 MONDO:0013412 should be listed under the AD   titinopathy    

Thank you!  Gioconda

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7346

add superclass to 'dilated cardiomyopathy 1G'
close https://github.com/monarch-initiative/mondo/issues/7347

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -243789,8 +243789,10 @@ xref: OMIM:604145 {source="MONDO:equivalentTo", source="DOID:0110430"}
 xref: UMLS:C1858763 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:604145"}
 is_a: MONDO:0016191 ! qualitative or quantitative defects of titin
 is_a: MONDO:0016333 {source="MONDO:Redundant", source="OMIM:604145"} ! familial dilated cardiomyopathy
+is_a: MONDO:0100494 {source="https://orcid.org/0000-0001-9310-0163"} ! autosomal dominant titinopathy
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:243792]: Why was this parent term selected vs. `MONDO:0100175` TTN-related myopathy?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -292394,10 +292396,12 @@ xref: UMLS:C1861065 {source="OMIM:613765", source="MONDO:equivalentTo", source="
 is_a: MONDO:0005045 {source="DC-OMIM:613765", source="DOID:0110315", source="MESH:C566044/inferred", source="MONDO:Redundant", source="OMIM:613765"} ! hypertrophic cardiomyopathy
 is_a: MONDO:0016191 ! qualitative or quantitative defects of titin
 is_a: MONDO:0024573 {source="MESH:C566044", source="MONDO:OMIM", source="OMIM:613765"} ! familial hypertrophic cardiomyopathy
+is_a: MONDO:0100494 {source="https://orcid.org/0000-0001-9310-0163"} ! autosomal dominant titinopathy
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:292399]: Why was this parent term selected vs. `MONDO:0100175` TTN-related myopathy?

**Review by twhetzel — COMMENTED** [review]
questions inline

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:243792]: that is what the user requested

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:243792]: oh wait, you are right, I am wrong!

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:292399]: my bad, I will fix

Comment (twhetzel) [reply src/ontology/mondo-edit.obo:292399]: Oh, so 'AD' there was autosomal dominant.... I did not get that until now. Thank you!!!

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**